### PR TITLE
[TM] Removes text() proc uses

### PR DESCRIPTION
**Pull request by Vi3trice**
## What Does This PR Do
Takes all the text() uses in the codebase and replace them with standard variable embedding.

## Why It's Good For The Game
Completes #21265.

## Testing
I checked the cases that were more complex than just moving a variable around, like Honkbot, the emergency shuttle, and whatever unholy thing that is AI alerts. Still needs a TM just in case because some places felt like someone just checked out their sanity at the door.

## Changelog
N/A

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -270,7 +270,7 @@
 		if(!lastbang)
 			lastbang = 1
 			for(var/mob/M in hearers(src, null))
-				to_chat(M, text("<FONT size=[]>BANG, bang!</FONT>", max(0, 5 - get_dist(src, M))))
+				to_chat(M, "<FONT size=[max(0, 5 - get_dist(src, M))]>BANG, bang!</FONT>")
```
Comment (Contrabang) [inline code/game/objects/structures/crates_lockers/closets.dm:273]: While you're here
```suggestion
				to_chat(M, "<font size=[max(0, 5 - get_dist(src, M))]>BANG, bang!</font>")
```

File: code/modules/assembly/proximity.dm [hunk 2]
```diff
@@ -103,7 +103,7 @@
 		return FALSE
 	var/second = time % 60
 	var/minute = (time - second) / 60
-	var/dat = text("<TT><B>Proximity Sensor</B>\n[] []:[]\n<A href='?src=[UID()];tp=-30'>-</A> <A href='?src=[UID()];tp=-1'>-</A> <A href='?src=[UID()];tp=1'>+</A> <A href='?src=[UID()];tp=30'>+</A>\n</TT>", (timing ? "<A href='?src=[UID()];time=0'>Arming</A>" : "<A href='?src=[UID()];time=1'>Not Arming</A>"), minute, second)
+	var/dat = "<TT><B>Proximity Sensor</B>\n[timing ? "<A href='?src=[UID()];time=0'>Arming</A>" : "<A href='?src=[UID()];time=1'>Not Arming</A>"] [minute]:[second]\n<A href='?src=[UID()];tp=-30'>-</A> <A href='?src=[UID()];tp=-1'>-</A> <A href='?src=[UID()];tp=1'>+</A> <A href='?src=[UID()];tp=30'>+</A>\n</TT>"
```
Comment (Contrabang) [inline code/modules/assembly/proximity.dm:106]: general comment, this is an unholy abomination

**Review by Contrabang — COMMENTED** [review]
Some minor stuff, looks pretty good. I would normally request that the lines you touched that have uppercase HTML tags be converted to lowercase, but that feels very out of scope here

**Review by Contrabang — APPROVED** [review]
Actually, no changes are really needed. LGTM

**Review by S34NW — APPROVED** [review]

**Comment by S34NW** [comment]
No reported issues, lets go

**Pull request merged**